### PR TITLE
Fea/removing queue and fixing test and config

### PR DESCRIPTION
**Pull request by fede-rodes**

### CODE REVIEW

File: contracts/RenPool.sol [hunk 1]
```diff
@@ -47,78 +46,41 @@ contract RenPool {
...
-
-            user.transfer(_amount);
-        }
-        else{
-            // Pool is locked, withdraw will be put in the queue
-            withdrawRequests.push(WithdrawRequest(user, _amount))
-
-        }
+        require(senderBalance > 0 && senderBalance >=  _amount);
+        require(isLocked == false, "Pool is locked");
```
Comment (fede-rodes) [inline contracts/RenPool.sol:74]: missing error message

File: scripts/deploy.py [hunk 2]
```diff
@@ -1,22 +1,24 @@
-from brownie import accounts, RenToken, RenPool
+from brownie import accounts, RenToken, RenPool, constants
 # ^ accounts is coming from Ganache while RenToken and RenPool from the contracts folder.
```
Comment (fede-rodes) [inline scripts/deploy.py:2]: remove constants

File: tests/test_ren_pool.py [hunk 3]
```diff
@@ -1,46 +1,63 @@
...
+FAUCET_AMOUNT = 1000 * 10 ** DECIMALS
+# ^ TODO: try to move these constants to a constants file so that we can import them
+
+def test_ren_mint(owner, ren_token):
     """
     Test REN tokens are properly minted.
     """
-    assert ren_token.balanceOf(accounts[0]) == 1e21
+    INITIAL_SUPPLY = 1000_000_000 * 10 ** DECIMALS
+    assert ren_token.balanceOf(owner) == INITIAL_SUPPLY
```
Comment (fede-rodes) [inline tests/test_ren_pool.py:11]: move INITIAL_SUPPLY up

File: tests/conftest.py [hunk 4]
```diff
@@ -10,15 +13,22 @@ def setup(fn_isolation):
...
 
 @pytest.fixture(scope="module")
-def ren_token(accounts, ERC20):
+def owner(accounts):
     """
-    Yield a `ERC20` object for the RenPool contract.
+    Yield an `Account` object for the owner of the contracts.
     """
-    yield ERC20.deploy("REN", "REN", 18, 1e21, {'from': accounts[0]})
+    yield accounts[0]
```
Comment (fede-rodes) [inline tests/conftest.py:21]: shall we introduce admin as well?

**Review by JonathanStefanov — APPROVED** [review]

**Pull request merged**